File: core/src/main/java/fr/sncf/osrd/infra_state/implementation/standalone/StandaloneSimulationError.java
package fr.sncf.osrd.infra_state.implementation.standalone;

import fr.sncf.osrd.exceptions.OSRDError;
import java.io.Serial;

public class StandaloneSimulationError extends OSRDError {
    @Serial
    private static final long serialVersionUID = -2966725866296617520L;
    public static final String osrdErrorType = "standalone_simulation";

    protected StandaloneSimulationError(String message) {
        super(message, ErrorCause.INTERNAL);
    }
}

File: core/src/main/kotlin/fr/sncf/osrd/dyn_infra/implementation/standalone/StandaloneSimulationError.kt
package fr.sncf.osrd.dyn_infra.implementation.standalone

import fr.sncf.osrd.exceptions.OSRDError
import java.io.Serial

open class StandaloneSimulationError protected constructor(message: String) : OSRDError(message, ErrorCause.INTERNAL) {
    companion object {
        @Serial
        const val serialVersionUID = -2966725866296617520L
        const val osrdErrorType = "standalone_simulation"
    }
}